src/main/java/com/synopsys/integration/alert/provider/blackduck/BlackDuckProviderContentTypes.java

```diff
@@ -165,6 +184,10 @@
 
     public static final List<ProviderContentType> ALL = Arrays.asList(BOM_EDIT, LICENSE_LIMIT, POLICY_OVERRIDE, RULE_VIOLATION, RULE_VIOLATION_CLEARED, VULNERABILITY);
 
+    private static JsonPath createJsonPath(final String pattern, final String... fields) {
```

Comment:
I believe this should be a public static method on JsonField because that is the thing that demands JsonPath (and providers other than Black Duck will likely need access to the it).

Reply:
I agree with Gavin here.  This method should be in JsonField

Reply:
Moved to JsonField